src/drivers/boards/px4fmu-v2/px4fmu2_init.c

```diff
@@ -212,19 +212,6 @@ static struct sdio_dev_s *sdio;
 
 #include <math.h>
 
-/* TODO XXX commented this out to get cmake build working */
-/*#ifdef __cplusplus*/
-/*__EXPORT int matherr(struct __exception *e)*/
-/*{*/
-/*return 1;*/
-/*}*/
-/*#else*/
-/*__EXPORT int matherr(struct exception *e)*/
-/*{*/
-/*return 1;*/
-/*}*/
-/*#endif*/
-
```

Comment:
Please keep this in separate commits. I don't see how this is related to the sensors.